contracts/minter/src/contract.rs

```diff
@@ -68,6 +68,17 @@ pub fn instantiate(
         });
     }
 
+    // Limit per address limit to 1% of num tokens
+    if (msg.num_tokens > 100 && msg.num_tokens < 5000)
```

Comment:
i don't like this conditional because `msg.num_tokens <= 100` => `per_address_limit` could be `50`.
maybe use ceil(dec(1/100))? i'm not sure